src/Talismans.ts

```diff
@@ -136,15 +137,14 @@ export const buyTalismanResources = (type: keyof typeof talismanResourceCosts, p
 export const showTalismanEffect = (i: number) => {
     DOMCacheGetOrSet("talismanlevelup").style.display = "none"
     DOMCacheGetOrSet("talismanEffect").style.display = "block"
-    DOMCacheGetOrSet("talismanrespec").style.display = "none"
-    const a = DOMCacheGetOrSet("talismanSummary")
-    const b = DOMCacheGetOrSet("talismanBonus")
-    const c = DOMCacheGetOrSet("talismanRune1Effect")
-    const d = DOMCacheGetOrSet("talismanRune2Effect")
-    const e = DOMCacheGetOrSet("talismanRune3Effect")
-    const f = DOMCacheGetOrSet("talismanRune4Effect")
-    const g = DOMCacheGetOrSet("talismanRune5Effect")
-    const h = DOMCacheGetOrSet("talismanMythicEffect")
+    const a = document.getElementById("talismanSummary")
+    const b = document.getElementById("talismanBonus")
+    const c = document.getElementById("talismanRune1Effect")
+    const d = document.getElementById("talismanRune2Effect")
+    const e = document.getElementById("talismanRune3Effect")
+    const f = document.getElementById("talismanRune4Effect")
+    const g = document.getElementById("talismanRune5Effect")
+    const h = document.getElementById("talismanMythicEffect")
```

Comment:
```suggestion
    const a = document.getElementById("talismanSummary")
    const b = document.getElementById("talismanBonus")
    const c = document.getElementById("talismanRune1Effect")
    const d = document.getElementById("talismanRune2Effect")
    const e = document.getElementById("talismanRune3Effect")
    const f = document.getElementById("talismanRune4Effect")
    const g = document.getElementById("talismanRune5Effect")
    const h = document.getElementById("talismanMythicEffect")
```

use `DOMCacheGetOrSet`